### DISCUSSION TITLE
Turn off all borders

### DISCUSSION THREAD

**Discussion by githorse** [KevinVandy/material-react-table#520] [Ideas]
Is there an (easy) way to turn off all outer borders around the grid?

Motivation: I often want seamless embedded grids (sometimes inside the master detail of an outer grid) that are really more like simple tables without much styling. See also [hiding the header row](https://github.com/KevinVandy/material-react-table/discussions/519).

**Comment by joey-kwl** [comment]
Are you talking about the "border" around the table? If so, they are shadows which you can use: 
```
muiTablePaperProps={{
	sx: {
		boxShadow: "unset",
		backgroundImage: "unset"
	}
}}
```  

to remove them.

If you want to remove border for each Cell, you can use: 
```
muiTableBodyCellProps={{
	sx: {
		border: "none"
	}
}}
```

**Reply by githorse** [reply]
This works. Might be worth having a prop for this (`enableBorders`) to make it more clear.

**Reply by KevinVandy** [reply]
I expose the paper props for this so won't be making another prop that does the same thing
